src/Umbraco.Core/PropertyEditors/Validation/ITypedJsonValidator.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+using Umbraco.Cms.Core.Models.Validation;
+
+namespace Umbraco.Cms.Core.PropertyEditors.Validation;
+
```

Comment:
Could you add an XML header comment here?  I think it would be useful to explain what it's used for an why you created it (i.e. to avoid deserialization of the same value multiple times).